storage/cycler.go

```diff
@@ -66,7 +68,11 @@ func (c *Cycler) run(fileKey string, keep int, deletePackage func(fileKey string
 	}
 }
 
-func (c *Cycler) load() {
+func (c *Cycler) load(cyclerFileName string) {
+	if !helper.IsExistsPath(cyclerPath) {
+		helper.Exec("mkdir", "-p", cyclerPath)
```

Comment:
helper.MkdirP(cyclerPath)

and you must rebase master that has fixed CI.

Reply:
Thanks, fixed.